community-membership.md

```diff
@@ -0,0 +1,213 @@
+# Community membership
+
+**Note:** This document is in progress
+
+This doc outlines the various responsibilities of contributor roles in
+Operator Framework. The Operator Framework project is subdivided into subprojects.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| member | active contributor in the community | sponsored by 2 reviewers, multiple contributions to the project | Operator Framework GitHub org member. |
+| reviewer | review contributions from other members | history of review and authorship in a subproject | [OWNERS] file reviewer entry. |
+| approver | approve accepting contributions | highly experienced and active reviewer + contributor to a subproject | [OWNERS] file approver entry|
+| subproject owner | set direction and priorities for a subproject | demonstrated responsibility and excellent technical judgment for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
+
+## New contributors
+
+New contributors should be welcomed to the community by existing members,
+helped with PR workflow, and directed to relevant documentation and
+communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the
+principles in this document, familiarity with project organization, roles,
+policies, procedures, conventions, etc., and technical and/or writing ability.
+Role-specific expectations, responsibilities, and requirements are enumerated
+below.
+
+## Member
+
+Members are continuously active contributors in the community. They can have
+issues and PRs assigned to them and pre-submit tests are automatically run for
+their PRs. Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the Operator Framework GitHub organization
+
+### Requirements
+
+- Enabled [two-factor authentication] on their GitHub account.
+- Have made multiple contributions to the project or community. Contribution may include, but is not limited to:
```

Comment:
```suggestion
- Have made multiple contributions to the project. A contribution may include, but is not limited to:
```

Reply:
Cannot the `OR community` make it questionable?

Reply:
I think the emphasis is meant to indicate that contributions outside of Github can count -- organizing a set of OF workshops, for example.